src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__big"
+      data-qa="big"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class `search--form__big` suggests a style rather than the function or content. Consider renaming it to reflect its purpose or content.